project.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "blues-stack",
```

Comment:
Is this supposed to be globally unique? As people use this template for their different projects, will they need to change this name? We can do this automatically in the `remix.init/index.js` file.